docker/scripts/lib_validation_testing.py

```diff
@@ -362,6 +362,34 @@ def runBenchmarkScoreScript(script=None,          # Script to run
 
 # End: def runBenchmarkScoreScript()
 
+def runDSDeepMarkScript(sourceDir=None,
+                    script=None,          # Script to run
+                   logID='',
+                   ompNumThreads=None,
```

Comment:
instead of setting `None` here, you can set them to default values.

Reply:
I will make a enhance in the next PR to handle the valid input.